cmd/flags/auth.go

```diff
@@ -81,19 +80,10 @@ const (
 )
 
 const (
-	authOIDCCAKeyFileFlag     = "auth-oidc-ca-key"
```

Comment:
why are you removing this? How can your e2e work? You should only remove the password